winml/adapter/FeatureDescriptorFactory.cpp

```diff
@@ -429,40 +519,43 @@ CreateImageFeatureDescriptor(
   const auto& tensor_type = type_proto.tensor_type();
   auto shape = WinML::ConvertShapeProtoToVector(tensor_type.shape());
   auto kind = WinML::TensorKindFromOnnxDataType(
-      onnx::TensorProto_DataType(tensor_type.elem_type()));
+  onnx::TensorProto_DataType(tensor_type.elem_type()));
 
   // pixel format and alpha
   auto pixel_format_value = FetchMetadataValueOrNull(metadata, c_bitmap_pixel_format_key);
   auto format_info = CreateBitmapPixelFormatAndAlphaModeInfo(pixel_format_value);
   auto pixel_format = format_info.first;
   auto alpha_mode = format_info.second;
 
+  // paulm:   commenting this out during layering.    gamma and nominal are never used
+  // since we only support one of them.  if a non support one is set, they all fall back 
+  // to TensorFeatureDescriptor (invalid image metadata)
+#ifdef DONE_LAYERING
```

Comment:
do we need to clean this up first?

Reply:
yeah, put this on this list.    we should clean it up , bare minimum we just delete the code.   but i wanted to take more time to think it through on how we want this code path to work .. what happehns, what we want to happen, and how we want to approach tensorization moving fwd.

---
In reply to: [354038577](https://github.com/microsoft/onnxruntime/pull/2534#discussion_r354038577) [](ancestors = 354038577)